e2e/test/scenarios/question-reproductions/reproductions-3.cy.spec.js

```diff
@@ -844,6 +848,36 @@ describe("issue 44415", () => {
   });
 });
 
+describe("issue 37374", () => {
```

Comment:
I managed to sort of reproduce the bug (the refresh button is not shown, but query gets auto-refreshed and ends up in a non-recoverable permission error state, like in original report)

https://github.com/metabase/metabase/assets/6830683/e78a2fce-c494-4cf0-9da4-546acdf1d0c9